posthog/api/person.py

```diff
@@ -64,8 +64,7 @@
 from posthog.queries.stickiness import Stickiness
 from posthog.queries.trends.lifecycle import Lifecycle
 from posthog.queries.util import get_earliest_timestamp
-from posthog.rate_limit import DestroyClickhouseModelThrottle
-from posthog.settings import EE_AVAILABLE, RATE_LIMIT_ENABLED
+from posthog.settings import EE_AVAILABLE
```

Comment:
nit: Use `from django.conf import settings` to get settings. This ensures you can easily override settings in tests, is the recommended way, and means you don't have to search for the right file for the setting when using a folder-style settings config

Reply:
oh, this is unrelated to current changes